tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -850,6 +851,39 @@ func printActionDeleted(entityName string) {
             }))
 }
 
+// Check if the specified action is a web-action
+func IsWebAction(client *whisk.Client, qname *QualifiedName) bool {
```

Comment:
I had it upper cased to export that function across packages, but currently, it's only used within the same package, so it can begin with lower case.